spec/services/user_invite_service_spec.rb

```diff
@@ -94,7 +94,7 @@
       )
     }.to change(ActionMailer::Base.deliveries, :count).by(1)
 
-    expect(result.reload.name).to eq("Name Not Provided")
+    expect(result.display_name).to eq("Name Not Provided")
```

Comment:
Hmm.   Assuming there was a reason there was a reload in the sequence in the first place,  I would expect to still have the reload in the changed spec.